Missionframework/modules/11_permission/fnc/fn_permission_export.sqf

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+/*
+    KPLIB_fnc_permission_export
```

Comment:
Is this function even used somewhere? 🤔 
And what exactly is the purpose?

Reply:
So basically before running a new map you can export the permission list and import them on the new map.
Or export/import them between different servers/communities/whatever.

Reply:
Already implemented and I've overlooked the usage?
The function currently just gets the permissionList and returns it. Where is that return used or saved to profileNamespace etc.?

Reply:
currently the list is returned and needs to get passed as argument for import

Reply:
Summary of the small discord chat:
Speaking of the "transport possibility" and "usage of the export function for transport and use it somewhere else as argument for the import".